tip-wlan/charts/wlan-ssc-service/files/cqlshrc.tip-wlan

```diff
@@ -67,12 +67,10 @@ color = on
 ;; A version of CQL to use (this should almost never be set)
 ; version = 3.2.1
 
-
-
 [connection]
 
 ;; The host to connect to
-hostname = tip-wlan-cassandra-headless
+hostname = {{ include "cassandra.service" . }}
```

Comment:
My only doubt is if Helm will be able to over-write this variable if we are passing this file as a template to the configMap. Not tried before but looks like a good way.

Reply:
Not sure if I understand your comment correctly. Helm will replace this variable with the correct service name as it will copy the file into the ConfigMap. That's what we need to do to be able to achieve a multi namespace deployment.

Reply:
Yes that was my comment. Thanks, looks good.